ReleaseNotes-2.9

```diff
@@ -63,6 +69,14 @@ Bugfixes:
  * Deletion of groups with users was not working
  * Fixed generation of the breadcrumb for the Image Editor (Kiwi)
  * Using ActiveSupport::JSON as encoder everywhere prevent to fail on Notification creation with events that have a very big payload
+ * Changes have been reduced in the Kiwi XML layout when modifications are made via the Kiwi Editor
```

Comment:
The editor did no exist before, so Editor is the feature, but we don't need to explain changes during the 2.9 development phase.

Reply:
Ok. Lines not needed removed.